packages/node_modules/@cerebral/inferno/package.json

```diff
@@ -14,17 +14,21 @@
   },
   "homepage": "http://cerebral.github.io/cerebral-website",
   "devDependencies": {
-    "babel-plugin-inferno": "^3.2.0"
-  },
-  "dependencies": {
-    "cerebral": "next"
+    "babel-plugin-inferno": "^3.2.0",
+    "cerebral": "^3.6.0-1511263649330",
+    "function-tree": "^2.2.1",
+    "inferno": "^3.10.1",
+    "inferno-component": "^3.10.1",
+    "typescript": "^2.6.2"
   },
+  "dependencies": {},
   "scripts": {
     "build": "cross-env BABEL_ENV=production babel src/ --out-dir=lib/ -s",
     "coverage": "nyc --reporter=lcov --reporter=json npm run test",
     "prepublish": "npm run build",
     "test": "mocha -r test/setup --compilers js:babel-register \"src/**/*.test.js\"",
-    "test:watch": "npm run test -- --watch"
+    "test:watch": "npm run test -- --watch",
+    "test:typings": "tsc -p tsconfig.json"
```

Comment:
Please look at how typings are run from @cerebral/react package. We like to run them along with all the other tests.